### ISSUE TITLE
Guide: Add Custom Encoder/Decoder guide

### ISSUE THREAD

**Issue by Andrew-Lees11** [IBM-Swift/kitura.io#284]
In our blog post [here](https://developer.ibm.com/swift/2018/09/01/kitura-custom-encoders-and-decoders/), we describe how to customise you Encoder and Decoder for the Kitura server. This would be useful to translate into a guide for the new learn page.

**Comment by ianpartridge** [comment]
I'm in two minds about this. While more docs are always a good idea, customising your encoder/decoder is a fairly niche thing to do...

**Comment by ddunn2** [comment]
If it's something we are saying is possible then we should provide some guidance on how you actually achieve it. 
I'll agree this probably isn't as high a priority as some of the other features we're missing guides for.

**Comment by Andrew-Lees11** [comment]
I agree this is fairly low priority and hopefully is someone wants to do it they would be able to find the blog post/message us in slack. I raised it because we already have the content ready to go so it wouldn't be too much work to migrate it into the new guide page (When we have a section for it)

**Comment by ddunn2** [comment]
I agree there should be an issue for this, and we should consider raising issues for anything else we find that we are missing. 
We want to make the docs as 'complete' as possible, so people do not need to search for our blogs or have to message us in Slack :)

If you have the content ready to go, you can raise a PR for it and add the doc here: https://github.com/IBM-Swift/kitura.io/tree/master/docs/unassigned-docs

Then later when we add the new section we can move this guide to it. 

Up to you though!
